### PR TITLE
[zh] sync /tasks/observability/logs/telemetry-api/

### PR DESCRIPTION
**Pull request by windsonsea**
Add zh text to this page:

```
content/zh/docs/tasks/observability/logs/telemetry-api/
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/observability/logs/telemetry-api/index.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
+---
+title: 使用 Telemetry API 配置访问日志
+description: 此任务向您演示如何使用 Telemetry API 配置 Envoy 代理来发送访问日志。
+weight: 10
+keywords: [telemetry,logs]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: yes
+---
+
+Telemetry API 如今在 Istio 中作为一流 API 已经有一段时间了。
```
Comment (zirain) [inline content/zh/docs/tasks/observability/logs/telemetry-api/index.md:10]:
```suggestion
Telemetry API 如今在 Istio 中作为一流 API 已经有一段时间了。
```

First Class API 不能这么直译，翻译成推荐感觉更好

cc @rootsongjc 有想法没？

File: content/zh/docs/tasks/observability/logs/telemetry-api/index.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+之前用户必须在 Istio 配置的 `MeshConfig` 部分配置遥测。
```
Comment (zirain) [inline content/zh/docs/tasks/observability/logs/telemetry-api/index.md:11]:
```suggestion
之前用户必须在 Istio 的 `MeshConfig` 中配置遥测。
```

File: content/zh/docs/tasks/observability/logs/telemetry-api/index.md (hunk 1 above)
Comment (kebe7jun) [reply content/zh/docs/tasks/observability/logs/telemetry-api/index.md:10]: 核心？ - 代表核心能力。

Comment (zirain) [reply content/zh/docs/tasks/observability/logs/telemetry-api/index.md:10]: 核心 感觉比推荐更好一些

**Review by zirain — APPROVED** [review]

**Pull request merged**